TF_Tooling_Web/src/Cursors/DotCursor.ts

```diff
@@ -38,22 +38,22 @@ export class DotCursor extends TouchlessCursor {
 
     // Function: constructor
     // Constructs a new cursor consisting of a central cursor and a ring.
-    // Optionally provide an _animationDuration to change the time it takes for the 'squeeze'
-    // confirmation animation to be performed. Optionally provide a _ringSizeMultiplier to change
-    // the size that the <cursorRing> is relative to the _cursor.
+    // Optionally provide an animationDuration to change the time it takes for the 'squeeze'
```

Comment:
🤷 The constructor variables could be layout out in the comment nicer than just a paragraph (e.g. bullet points)